### ISSUE TITLE
Android TextInput auto Dismiss Keyboard 

### ISSUE THREAD

**Issue by Sean-Snow** [facebook/react-native#16563]
### Is this a bug report?

yes

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?

yes

### Environment
    OS:  macOS Sierra 10.12.6
    Node:  9.2.0
    Yarn:  1.3.2
    npm:  5.5.1
    Watchman:  4.7.0

  Packages: (all installed)
```
   "accounting": "^0.4.1",
    "babel-polyfill": "^6.26.0",
    "fs": "0.0.2",
    "jcore-react-native": "^1.2.3",
    "jpush-react-native": "^2.1.7",
    "lokijs": "^1.5.1",
    "md5": "^2.2.1",
    "moment": "^2.20.1",
    "object-path": "^0.11.4",
    "prop-types": "^15.6.0",
    "react": "16.2.0",
    "react-native": "0.52.1",
    "react-native-animatable": "^1.2.4",
    "react-native-code-push": "^5.2.1",
    "react-native-svg": "6.0.1-rc.3",
    "react-navigation": "^1.0.0-beta.27",
    "react-redux": "^5.0.6",
    "reactnativecomponents": "^0.0.3-alpha.35",
    "redux": "^3.7.2",
    "redux-observable": "^0.17.0",
    "rxjs": "^5.5.6",
```

### Steps to Reproduce
1. TextInput get Focus

### Expected Behavior
I hope the keyboard is always there

### Actual Behavior
The keyboard dismiss automatically after a few seconds

### Reproducible Demo
```js
<View>
<TextInput/>
</View>
```

**Comment by jackple** [comment]
Is there some other way you try to dismiss keyboard?

**Comment by Sean-Snow** [comment]
no

**Comment by syq7970** [comment]
@Sean-Snow I have the same issule, how did you resolve this?

**Comment by Sean-Snow** [comment]
@syq7970 There is no solution

**Comment by kamal** [comment]
@syq7970 @Sean-Snow my workaround on 0.49.3 is to focus and then blur immediately on Android.

```typescript
componentDidMount () {
  if (Platform.OS === 'android') {
    this.nameField.focus()
    this.nameField.blur()
  }
}
```


This seems to be fixed in 0.50.4. In fact, the workaround crashes Android due to this issue https://github.com/facebook/react-native/issues/16821

**Comment by burakggs** [comment]
Hi @Sean-Snow,
How did you resolve this problem ?
